tools/install.sh

```diff
@@ -113,28 +115,33 @@ function _omb_install_run {
 function _omb_install_banner {
   # MOTD message :)
   printf '%s' "$GREEN"
-  printf '%s\n' \
-    '         __                          __               __  ' \
-    '  ____  / /_     ____ ___  __  __   / /_  ____ ______/ /_ ' \
-    ' / __ \/ __ \   / __ `__ \/ / / /  / __ \/ __ `/ ___/ __ \' \
-    '/ /_/ / / / /  / / / / / / /_/ /  / /_/ / /_/ (__  ) / / /' \
-    '\____/_/ /_/  /_/ /_/ /_/\__, /  /_.___/\__,_/____/_/ /_/ ' \
-    '                        /____/                            .... is now installed!'
+  local BANNER
+  BANNER=$(cat <<-'EOF'
+	         __                          __               __  
+	  ____  / /_     ____ ___  __  __   / /_  ____ ______/ /_ 
+	 / __ \/ __ \   / __ `__ \/ / / /  / __ \/ __ `/ ___/ __ \
+	/ /_/ / / / /  / / / / / / /_/ /  / /_/ / /_/ (__  ) / / /
+	\____/_/ /_/  /_/ /_/ /_/\__, /  /_.___/\__,_/____/_/ /_/ 
+	                        /____/                            .... is now installed!
+EOF
+  )
```

Comment:
We are using `printf '%s\n'` to avoid the fork cost. By the way, is this really reported by `shellcheck`?

Reply:
Thanks so much for mentioning the point about `fork`. I would not have thought to considered that at all. I've updated the code to use pure Bash following the advice I found [here](https://serverfault.com/a/72511), [here](https://serverfault.com/questions/72476/clean-way-to-write-complex-multi-line-string-to-a-variable#comment497435_72511) and [here](https://serverfault.com/questions/72476/clean-way-to-write-complex-multi-line-string-to-a-variable#comment579993_72511). However, this is approaching code that might be a bit too "clever," so I'd love to hear what you think.

Reply:
This is my personal preference, but I would like to avoid also the use of here documents if it's not necessary because the here documents actually save the contents to temporary files. The recent versions of Bash (Bash 5.1 and higher) started to create a pipe instead of a temporary file for a here document whose content is not larger than 64 KiB (or some other values depending on the operating system). But, the creation of pipe seems still redundant to me compared to the arguments to `printf '%s\n'`. If the reason to switch is just because of the false positive of shellcheck but not because it is better than `printf '%s\n'`, I wouldn't like to switch honestly. Do you have better reasoning?

Reply:
For sure, after learning that, I can definitely agree that using a heredoc would be overkill. One thing I came across, however, is that `# disable shellcheck=SC1003,SC2016` above the call to `printf` does not seem to silence the error. Given that, would it be appropriate to split up this block into separate `printf` command lines in order to both maintain visual consistency and be able to add a single ShellCheck ignore comment above the offending line containing backticks?

Reply:
Thank you for updating the commit!

> however, is that even putting `# disable shellcheck=SC1003,SC2016` above the call to `printf` does not seem to silence the error.

Isn't that a typo of `# shellcheck disable=SC1003,SC2016` (I mean `disable` and `shellcheck` are swapped). As far as I try in https://www.shellcheck.net/, it does seem to suppress the errors.

Reply:
Oh dear, I have no idea how I made that mistake. Thanks so much for catching that! Updating the PR now...

Reply:
Done!